refinery/tool_manager/models.py

```diff
@@ -429,6 +443,18 @@ def launch(self):
         return JsonResponse({Tool.TOOL_URL: self.get_relative_container_url()})
 
 
+def handle_bioblend_exceptions(func):
```

Comment:
May need to be reworked, but also may not. Could probably just decorate `launch()` since these bioblend calls happen in its scope